src/Connector.SqlServer/Connector/Container.cs

```diff
@@ -2,24 +2,33 @@
 using System.Linq;
 using CluedIn.Core.Connectors;
 using CluedIn.Core.Data.Vocabularies;
+using CluedIn.Core.Streams.Models;
 
 namespace CluedIn.Connector.SqlServer.Connector
 {
     public class Container
     {
-        private static readonly ConnectionDataType[] _codeEdgeColumns = new[]
-        {
-            new ConnectionDataType { Name = "OriginEntityCode".SqlSanitize(), Type = VocabularyKeyDataType.Text },
-            new ConnectionDataType { Name =  "Code".SqlSanitize(), Type = VocabularyKeyDataType.Text }
+        private static readonly ConnectionDataType[] _codeEdgeColumns = {
+            new ConnectionDataType { Name = "OriginEntityCode", Type = VocabularyKeyDataType.Text },
+            new ConnectionDataType { Name =  "Code", Type = VocabularyKeyDataType.Text }
         };
 
-        public Container(string containerName)
+        private static readonly ConnectionDataType[] _eventStreamCodeEdgeColumns = {new ConnectionDataType {Name = "CorrelationId", Type = VocabularyKeyDataType.Text},};
+
+        public Container(string containerName, StreamMode mode)
```

Comment:
Breaking change - Keep the original constructor, calling this new on with the default mode (Steam.Sync?)
Then mark the original constructor as obsolete.